tests/ut-gamemaker.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import gamemaker
+import database
+import team
+import user
+
+class TestMakerLogin(unittest.TestCase):
+  def setUp(self):
+    self.database = database.Database()
+    self.maker1 = gamemaker.GameMaker()
+  
+  def test_BadLogin(self):
+     self.assertEquals(self.maker1.login("Notvalidusername","Notvalidpassword"), "Bad username or password!", "Bad response to invalid login")
```

Comment:
Should be written as:

def test_bad_login(self):
    self.maker1.login()
    assertEquals(self.database.curUser, self.maker1, "Error Message")

Same for tests below